src/App.vue

```diff
@@ -162,10 +192,10 @@ export default {
         if (this.$mixpanel.get_distinct_id() != this.user.id.toString()) {
           this.$mixpanel.alias(this.user.id.toString());
           this.$mixpanel.people.set({
-            "$first_name": this.user.first_name,
-            "$last_name": this.user.last_name,
-            "$email": this.user.email,
-            "$phone": this.user.phone,
+            $first_name: this.user.first_name,
+            $last_name: this.user.last_name,
+            $email: this.user.email,
+            $phone: this.user.mobile,
```

Comment:
ye $first_name and other variables, linter ki wajah se fatt gaya

Reply:
App.vue mein, turn off the auto save feature of the linter (from the bottom bar), edit this, save this, commit the change, then turn the auto save feature on. I do this only for App.vue